files/XENONnT_wfsim_config.json

```diff
@@ -8,6 +8,7 @@
 {
     // Configuration flags
     "s1_model_type": "simple",
+    "s2_time_model": "s2_time_spread around zero",
```

Comment:
Isn't it better if the default is `'optical_propagation'` ?
We also set that for the sr0 config [here](https://github.com/XENONnT/private_nt_aux_files/blob/02edbf6ba2e977aabc5e412d8558864ddd9497d4/sim_files/fax_config_nt_sr0_v0.json#L25)

Reply:
Hi, This part of [the test](https://github.com/XENONnT/WFSim/blob/19dde4b1a7b833cf41c823c0a4fecfad5f386b14/tests/test_wfsim.py#L62) won't be loading any data from our database, so we can't use optical propagation. While the [test_sim_nT_advanced](https://github.com/XENONnT/WFSim/blob/19dde4b1a7b833cf41c823c0a4fecfad5f386b14/tests/test_wfsim.py#L104) will use sr0 config and thus also test the optical map. And this way we can also maximize test coverage.